src/main/java/bio/terra/cli/businessobject/Workspace.java

```diff
@@ -331,8 +339,12 @@ public String grantBreakGlass(
 
   // ====================================================
```

Comment:
Can we get rid of this decoration? I know what accessors and mutators are.

Reply:
Done